lerobot/common/envs/utils.py

```diff
@@ -39,7 +39,7 @@ def preprocess_observation(observations: dict[str, np.ndarray]) -> dict[str, Ten
 
             # sanity check that images are channel last
             _, h, w, c = img.shape
-            assert c < h and c < w, f"expect channel first images, but instead {img.shape}"
+            assert c < h and c < w, f"expect channel last images, but instead {img.shape}"
```

Comment:
Thanks! Just a suggestion to make this message even better

```suggestion
            assert c < h and c < w, f"expect channel last images, but instead got {img.shape=}"
```